Public key export doesn't work

I was following our installation & usage steps from the tool's documentation and it looks like that the export function doesn't work on my mac. The exporting function crashes at:

```
hvge@trogdor:bin-tool$ java -jar ssl-pinning-tool.jar export -k keypair.pem
2018-09-11 21:30:35 - SEVERE Failed to convert private key, error: sun.security.ec.ECPrivateKeyImpl cannot be cast to org.bouncycastle.jce.interfaces.ECPrivateKey
com.wultra.security.ssl.pinning.errorhandling.SSLPinningException: Failed to convert private key, error: sun.security.ec.ECPrivateKeyImpl cannot be cast to org.bouncycastle.jce.interfaces.ECPrivateKey
	at com.wultra.security.ssl.pinning.Application.exportPublicKey(Application.java:451)
	at com.wultra.security.ssl.pinning.Application.executeCommand(Application.java:189)
	at com.wultra.security.ssl.pinning.Application.main(Application.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:51)
Caused by: java.lang.ClassCastException: sun.security.ec.ECPrivateKeyImpl cannot be cast to org.bouncycastle.jce.interfaces.ECPrivateKey
	at com.wultra.security.ssl.pinning.Application.exportPublicKey(Application.java:447)
	... 10 more
```

 I was following our own [BC installation steps](https://github.com/wultra/powerauth-server/wiki/Installing-Bouncy-Castle) and BouncyCastle is the last provider in the `java.security` file. 

Other variables:
- I just downloaded tool from official releases
- Java version 
   ```
   hvge@trogdor:bin-tool$ java -version
   java version "1.8.0_121"
  Java(TM) SE Runtime Environment (build 1.8.0_121-b13)
  Java HotSpot(TM) 64-Bit Server VM (build 25.121-b13, mixed mode)
  ```
- BC version: `bcprov-jdk15on-1.60.jar`

--- a/src/main/java/com/wultra/security/ssl/pinning/Application.java
+++ b/src/main/java/com/wultra/security/ssl/pinning/Application.java
@@ -308,7 +308,7 @@ public class Application {
         try (FileReader fileReader = new FileReader(privateKeyPath)) {
             final PEMParser pemParser = new PEMParser(new BufferedReader(fileReader));
             // Expected key type is EC
-            final KeyFactory kf = KeyFactory.getInstance("EC");
+            final KeyFactory kf = KeyFactory.getInstance("ECDSA", "BC");
             final Object pemInfo = pemParser.readObject();
             pemParser.close();
             if (pemInfo instanceof PrivateKeyInfo) {

